### DISCUSSION TITLE
Maintainers community

### DISCUSSION THREAD

**Discussion by sergey-tihon** [fsprojects/FsProjectsAdmin#36] [General]
Dear @fsprojects/maintainers 

During last weeks we cleaned up teams structure in fsprojects organisation:
1. Removed all teams used to grand access to one repo for 1 maintainer.
2. Created groups that share provide access to similar projects like @fsprojects/typeprovider-authors  & @fsprojects/fleece-team 
3. Created one group that contain all org members @fsprojects/maintainers  to facilitate communication.
4. See more here https://github.com/orgs/fsprojects/teams

Please take a look on new structure and leave comment here if:
1. You do not want to be a member of @fsprojects/maintainers  teams and receive all @ mention notification
2. You are no longer interested to be a member of fsproject org
3. You need more teams or have improvement suggestion

**Comment by dsyme** [comment]
> 2.	Created groups that share provide access to similar projects like @fsprojects/typeprovider-authors & @fsprojects/fleece-team

I'm not so sure about this - our policies don't say anything about, for example, all type provider authors getting mutual maintainer access. There are risks in having too many maintainers or people assigned maintainer role on things that don't expect.

When one project has several repos it makes sense. But I think we need to treat each project separately?

**Reply by sergey-tihon** [reply]
fsprojects/typeprovider-authors group servers two reasons
1. group people with expertise and create communication channel
2. group similar projects in organisation.

Write access provided only not actively maintained repos. For most repost access level is Read (mean nothing for OSS repos)

**Comment by dsyme** [comment]
OK, that's good. Could you add a couple of lines to the policies to mention that maintainers may be added to topic groups of shared interest, and can opt out. Perhaps also mention that they can can also participate by contributing or proposing new ones? Thanks :)

**Comment by LukeAllen** [comment]
Hey, I'm not able to contribute any more so I shouldn't be a member. Good luck!

**Reply by shishkin** [reply]
me too

**Comment by haf** [comment]
I’d be happy if you can let me go, too.

**Comment by Dzoukr** [comment]
Same here. 😞

**Reply by dmitry-a-morozov** [reply]
Hi, 
I don't have capacity to contribute. Please remove me from all `fsprojects` teams.

**Comment by Alxandr** [comment]
I've not used fsharp for years, so I don't think I should be part of any `fsprojects` teams either.

**Comment by dsyme** [comment]
Cool, we'll get things cleaned up. And thank you to those maintainers departing for their contributions to incubating projects :-)

**Reply by sergey-tihon** [reply]
@LukeAllen @shishkin @haf @Dzoukr  @dmitry-a-morozov @Alxandr
Done. I've removed you from `maintainers` team.

If you feel like you want to leave `fsprojects` organization as well, [you can do it anytime](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/removing-yourself-from-an-organization).

**Reply by Alxandr** [reply]
Thanks :)
